ocaml/xapi/xapi_globs.ml

```diff
@@ -951,7 +951,11 @@ let ignore_vtpm_unimplemented = ref false
 
 let evacuation_batch_size = ref 10
 
-type xapi_globs_spec_ty = Float of float ref | Int of int ref
+type xapi_globs_spec_ty =
+  | Float of float ref
+  | Int of int ref
+  | Timeout of Xapi_stdext_unix.Unixext.Timeout.t ref
```

Comment:
What is the difference between a span and a timeout? What are Float and Int denoting?

Reply:
Float and Int are the existing types for xapi globs entries. Timeout is the new type. Some ocamldoc comments would probably be useful, I'll add some.

Reply:
Not all the time spans in xapi globs are for timeouts, I believe TimeSpan would encompass periods as well.  
```
| TimeSpan of Mtime.Span.t ref
```

Reply:
I don't understand why Span and Timeout need different variants here either

Reply:
I should probably change `remaining ?attempt_delay` to take a `Timeout.t` argument, then I can drop `Span` from here.

Reply:
Rebased on master, there is now a short and long duration instead (and we only use the short duration).